README.md

```diff
@@ -1,602 +1,361 @@
-      expect(mail.body.encoded).to match("Hi")
-    end
-  end
-end
+RSpec.describe User, type: :model do
+...
 ```
 
-For more information, see the [cucumber scenarios for mailer specs
-](https://relishapp.com/rspec/rspec-rails/v/3-4/docs/mailer-specs).
-
-## Job specs
-
-Tagging a context with the metadata `:type => :job` treats its examples as job
-specs. Typically these specs will live in `spec/jobs`.
-
-```ruby
-require 'rails_helper'
-
-RSpec.describe UploadBackupsJob, :type => :job do
-  describe "#perform_later" do
-    it "uploads a backup" do
-      ActiveJob::Base.queue_adapter = :test
-      UploadBackupsJob.perform_later('backup')
-      expect(UploadBackupsJob).to have_been_enqueued
-    end
-  end
-end
-```
+[in the appropriate folder]: https://relishapp.com/rspec/rspec-rails/docs/directory-structure
 
-For more information, see the [cucumber scenarios for job specs
-](https://relishapp.com/rspec/rspec-rails/docs/job-specs).
+### System specs, feature specs, request specs–what’s the difference?
 
-## View specs
+RSpec Rails provides some end-to-end (entire application) testing capability
+to specify the interaction with the client.
 
-View specs default to residing in the `spec/views` folder. Tagging any context
-with the metadata `:type => :view` treats its examples as view specs.
+#### System specs
 
-View specs mix in `ActionView::TestCase::Behavior`.
+Also called **acceptance tests**, **browser tests**, or **end-to-end tests**,
+system specs test the application from the perspective of a _human client._
+The test code walks through a user’s browser interactions,
 
-```ruby
-require 'rails_helper'
+* `visit '/login'`
+* `fill_in 'Name', with: 'jdoe'`
 
-RSpec.describe "events/index", :type => :view do
-  it "renders _event partial for each event" do
-    assign(:events, [double(Event), double(Event)])
-    render
-    expect(view).to render_template(:partial => "_event", :count => 2)
-  end
-end
+and the expectations revolve around page content.
 
-RSpec.describe "events/show", :type => :view do
-  it "displays the event location" do
-    assign(:event, Event.new(:location => "Chicago"))
-    render
-    expect(rendered).to include("Chicago")
-  end
-end
-```
+* `expect(page).to have_text('Welcome')`
 
-View specs infer the controller name and path from the path to the view
-template. e.g. if the template is `events/index.html.erb` then:
+Because system specs are a wrapper around Rails’ built-in `SystemTestCase`,
+they’re only available on Rails 5.1+.
 
-```ruby
-controller.controller_path == "events"
-controller.request.path_parameters[:controller] == "events"
-```
+#### Feature specs
 
-This means that most of the time you don't need to set these values. When
-spec'ing a partial that is included across different controllers, you _may_
-need to override these values before rendering the view.
+Feature specs serve the same purpose,
+but `SystemTestCase` solves some longstanding configuration issues they had,
```

Comment:
It doesn't solve some longstanding configuration issues, it just has Rails handle it for you.